docs/tutorials/orc.ipynb

```diff
@@ -0,0 +1,333 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.        init,
</pre><p>this should be <code>vocab_init</code>  ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/io/pull/1465/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
right